docs/faq.rst

```diff
@@ -162,10 +162,18 @@ How can we reduce the airflow UI page load time?
 
 If your dag takes long time to load, you could reduce the value of ``default_dag_run_display_number`` configuration in ``airflow.cfg`` to a smaller value. This configurable controls the number of dag run to show in UI with default value 25.
 
+
 How to fix Exception: Global variable explicit_defaults_for_timestamp needs to be on (1)?
 ---------------------------------------------------------------------------------------------
 
 This means ``explicit_defaults_for_timestamp`` is disabled in your mysql server and you need to enable it by:
 
 #. Set ``explicit_defaults_for_timestamp = 1`` under the mysqld section in your my.cnf file.
 #. Restart the Mysql server.
+
+
+How to reduce airflow dag scheduling latency in production?
+------------------------------------------------------------
```

Comment:
Could you fix the length of this line? Otherwise it will throw an error when compiling the docs.

Reply:
I use online rst editor which the doc seems to render fine. My understanding is that as long as the length of line is longer than the txt, then we are fine?

Reply:
Got you :) pr updated.